### PR TITLE
Notifications update

### PR DESCRIPTION
**Pull request by architaharathi**
Updating the Notifications page to not have absolute styles and including a FlatList

Issue #53 

simulator screen shot - iphone xr - 2019-02-23 at 09 38 28
simulator screen shot - iphone xr - 2019-02-23 at 09 38 45

### CODE REVIEW

File: screens/Notifications.js [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
-import { StyleSheet, Text, View, Image, Dimensions } from 'react-native';
+import { StyleSheet, Text, View, Image, Dimensions, FlatList } from 'react-native';
 
 
 export default class Notifications extends React.Component {
   render() {
     return (
       <View style={styles.container}>
-        <View style={styles.orangeRectangle}></View>
+        <View style={styles.peachBox}></View>
```
Comment (ijemmao) [inline screens/Notifications.js]: `View` components that don't have content inside them, can just be closed like `<View />`

File: screens/Notifications.js [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
         <Text style={styles.header}>Notifications!</Text>
-        <View style={styles.whiteCard}></View>
-        <View style={styles.slantedBlueRectangle}></View>
-        <View style={styles.slantedPurpleRectangle}></View>
+        <View style={styles.iconsBox}>
+          <Image source={require('./../assets/images/top_shopping_cart.png')}style={styles.shopCart} />
+          <Image source={require('./../assets/images/profile_head.png')}style={styles.personIcon} />
+        </View>
+        <View style={styles.blueBox}></View>
```
Comment (ijemmao) [inline screens/Notifications.js]: `View` components that don't have content inside them, can just be closed like `<View />`

File: screens/Notifications.js [hunk 3]
```diff
@@ -1,19 +1,35 @@
...
+        <View style={styles.purpleBox}></View>
```
Comment (ijemmao) [inline screens/Notifications.js]: `View` components that don't have content inside them, can just be closed like `<View />`

File: screens/Notifications.js [hunk 4]
```diff
@@ -1,19 +1,35 @@
...
         <Text style={styles.recentOrdersText}>Your recent orders...</Text>
-        <Image source={require('./../assets/images/top_shopping_cart.png')}style={styles.topShopCart} />
-        <Image source={require('./../assets/images/profile_head.png')}style={styles.profileHead} />
+        <View style={styles.whiteBox}></View>
+          <FlatList style={{marginTop: -Dimensions.get("screen").height*.46}} vertical={true}
+            data={[{key: 'Notifcation # 1'}, {key: 'Notification #2' },
```
Comment (ijemmao) [inline screens/Notifications.js]: The move the `data` array out of inline and make it a variable. This current setup makes it harder to read!

File: screens/Notifications.js [hunk 5]
```diff
@@ -1,19 +1,35 @@
...
+              {item}) => {
+                return(
+              <View>
+                <Text style={styles.notificationList}>{item.key}</Text>
+              </View>
+               )
+              }
+            }
+          />
+          <View style={{marginTop: Dimensions.get("screen").height*.06}}></View>
```
Comment (ijemmao) [inline screens/Notifications.js]: Remove inline styles and `View` components that don't have content inside them, can just be closed like `<View />`

File: screens/Notifications.js [hunk 6]
```diff
@@ -64,41 +107,24 @@ const styles = StyleSheet.create({
...
+      justifyContent: 'space-between',
+      marginTop: -Dimensions.get('screen').height*0.1,
+      width: Dimensions.get('screen').width,
     },
-    topShopCart: {
-      position: 'absolute',
-      top: 50,
-      left: 20,
-      height: 40,
+    shopCart: {
```
Comment (ijemmao) [inline screens/Notifications.js]: Use the `variables` file for `shopCart` styles

File: screens/Notifications.js [hunk 7]
```diff
@@ -64,41 +107,24 @@ const styles = StyleSheet.create({
...
-    profileHead: {
-      position: 'absolute',
-      top: 50,
-      right: 20,
       height: 40,
+      marginTop: -Dimensions.get('screen').height*0.1,
+      justifyContent: 'flex-start',
+      marginLeft: 15,
+    },
+    personIcon: {
```
Comment (ijemmao) [inline screens/Notifications.js]: Use `variables` file for `personIcon` styles

**Comment by architaharathi** [comment]
simulator screen shot - iphone xr - 2019-02-24 at 11 43 34

This is how it looks after the changes I made!

**Comment by ijemmao** [comment]
Looking good 👍🏾

**Pull request merged**